docs/application-connector/08-02-get-client-certificate.md

```diff
@@ -161,5 +162,5 @@ curl https://gateway.kyma.local/{APP_NAME}/v1/metadata/services --cert {CERT_FIL
 Use this command to call the Event Service:
 
 ```bash
-curl https://gateway.kyma.local/{APP_NAME}/v1/events --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
+curl https://gateway.kyma.local/{APP_NAME}/v1/events/subscribed --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
```

Comment:
Shouldn't the `{CERT_FILE_NAME}` part be changed into `{CLIENT_CERT_FILE_NAME}` here as well?

Reply:
It can be, for clarification purposes. Thanks!